scripts/greetings.coffee

```diff
@@ -2,8 +2,14 @@ module.exports = (robot) ->
   enterReplies = require('../stubs/enterReplies')
   leaveReplies = require('../stubs/leaveReplies')  
   robot.enter (res) ->
-    res.send res.random enterReplies
+    console.log(robot.adapter.client.rtm.dataStore.getChannelGroupOrDMById res.message.room)
```

Comment:
is this related to the bump, or pork barrel?

Reply:
unrelated

Reply:
I'm cool with it in one PR, but can it have a separate commit?